### PR TITLE
Readme fixes.

### PR DESCRIPTION
**Pull request by juanjux**
Signed-off-by: Juanjo Alvarez <juanjo@sourced.tech>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,14 +28,16 @@ If the build fails because it can't find the `jni.h` header file, run it with:
 
 Changing the JDK directory to the one right for your system.
 
-*Note: currently only Linux is supported, check [#7](https://github.com/bblfsh/client-scala/issues/7) for MacOS support*.
+*Note: currently only Linux is supported, check
+[#7](https://github.com/bblfsh/client-scala/issues/7) for MacOS support*.
```
Comment (abeaumont) [inline README.md]: As commented, the _hash 7_ already generates the link for you =)

File: README.md [hunk 2]
```diff
@@ -44,7 +47,7 @@ for other systems just look at maven central's dependency information.
 </dependency>
 ```
 
-#### Scala SBT
+#### Scala' sbt
```
Comment (abeaumont) [inline README.md]: The quote shouldn't be there or there's a _s_ missing afterwards, I think.

File: README.md (hunk 1 above)
Comment (juanjux) [reply README.md]: Ups, true.

File: README.md (hunk 2 above)
Comment (juanjux) [reply README.md]: The `s` is omitted if the next letter is an `s` or sound like an `s`, but I'm not so sure about acronyms, I'll just remove it to be sure.

Comment (abeaumont) [reply README.md]: Hmmm, no, the next word is not considered for the omission of the _s_ after the apostrophe.

File: README.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ If the build fails because it can't find the `jni.h` header file, run it with:
 Changing the JDK directory to the one right for your system.
 
 *Note: currently only Linux is supported, check
-[#7](https://github.com/bblfsh/client-scala/issues/7) for MacOS support*.
+[#7] for MacOS support*.
```
Comment (abeaumont) [inline README.md]: The brackets are unnecessary too

**Pull request merged**